scripts/scr_bomb_world/scr_bomb_world.gml

```diff
@@ -171,13 +189,13 @@ function scr_bomb_world(star_system, planet_number, bombard_target_faction, bomb
     
 	    if (kill>0) then kill=min(star_system.p_population[planet_number],kill);
     
-	    txt3="";
+	    txt3=""; // Life is the Emperor's currency. Spend it well
 	    if (pop_before>0) and (star_system.p_type[planet_number]!="Daemon"){
 	        if (star_system.p_large[planet_number]==0){
 	        	pop_after=round(max(0,pop_after-kill));
-	        	txt3="##It had a civilian population of "+string(scr_display_number(floor(pop_before)))+" and "+string(scr_display_number(floor(kill)))+" die over the duration of the bombardment.";
+	        	txt3="##It had Imperium subject population of "+string(scr_display_number(floor(pop_before)))+" and "+string(scr_display_number(floor(kill)))+" were annihilated over the duration of the bombardment.";
 	    	}else if (star_system.p_large[planet_number]=1){
-	    		txt3="##It had a civilian population of "+string(pop_before)+" billion and "+string(kill)+" billion die over the duration of the bombardment.";
+	    		txt3="##It had Imperium subject population of "+string(pop_before)+" billion and "+string(kill)+" billion were annihilated over the duration of the bombardment.";
```

Comment:
Currently, all the planets are under theoretical control of the Imperium. We do not have alien factions who can truly say that "this is our world". We do not have T'au Worlds, Ork worlds, who have their own "civilian" (Imperium would not consider alien a civilian, I'm 99% certain, the few exceptions could be inquisitorial and Rogue trader retinues) populations.
Hence, the civilians in this game are humanoid imperial subjects by default.

Unless You are planning to implement new planets with alien populations? I can see interesting scenario where the player is on the offensive, "liberating" the sector, rather than defending it, as we do now.

Currently, adding this check would deceive the player - the game would state that "ork population" was reduced, yet the player could receive penalties for bombarding a Hive, Forge or Shrine world, which would create more confusion for new players.